.github/workflows/rust.yml

```diff
@@ -30,6 +30,25 @@ jobs:
   build_and_test:
     name: Rust and clippy tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        bash_docker_image:
+          - "bash:5.2"
+          - "bash:5.1"
+          - "bash:5.0"
+          - "bash:4.4"
+          - "bash:4.3"
+          - "bash:4.2"
+          - "bash:4.1"
+          - "bash:4.0"
+          - "bash:3.2"
```

Comment:
```suggestion
          # - "bash:4.1"
          # - "bash:4.0"
          # - "bash:3.2"
```

test_validity_src_tests_validity_array_get_index_by_copy_ab which is introduced at #628 fails for bash 3.2, 4.0 and 4.1.
Should we drop these versions also?